tests/core/web3-module/test_clientVersion.py

```diff
@@ -1,2 +0,0 @@
-def test_web3_clientVersion(w3):
-    assert w3.clientVersion.startswith("EthereumTester/")
```

Comment:
I may have just missed it, but it looks like this file got deleted instead of moved?

Reply:
Thanks for catching this! Just forgot to push up my latest branch where I renamed this file.